vmm/src/memory_manager.rs

```diff
@@ -47,6 +48,11 @@ const X86_64_IRQ_BASE: u32 = 5;
 
 const HOTPLUG_COUNT: usize = 8;
 
+// Memory policy constants
+const MPOL_BIND: u32 = 2;
+const MPOL_MF_STRICT: u32 = 1 << 0;
+const MPOL_MF_MOVE: u32 = 1 << 1;
+
```

Comment:
It is more like a question than a comment: for the syscalls that are missing a wraper in the `libc` crate, do we normally just define a wrapper function and related constants in the module we use it? 

Do you think it is worth adding a crate where we can define all the raw`syscalls` and related constants/flags in a unified place? Maybe we don't need it, as we don't use much raw `syscalls` in our code base (at least for now).

Reply:
There are multiple crates already doing such thing (`nix` is a big one), but we want to rely on `libc` only if possible, and avoid pulling big dependencies for just a few syscalls.
As you mentioned, there are only a few system call which need to be handled manually through our codebase so it's fine to have them in cloud-hypervisor for now.
Long term, we can expect `libc` to include more and more syscalls, which would reduce the debt on our side. The other option is to push the missing syscalls to the `vmm-sys-util` crate on `rust-vmm`.